src/main/resources/crafter/studio/upgrade/pipelines.yaml

```diff
@@ -548,6 +557,16 @@ configurations:
           - type: xsltFileUpgrader
             path: *permission-mappings-config
             template: crafter/studio/upgrade/update-version.xslt
+      - currentVersion: 4
+        nextVersion: 5
+        operations:
+          - type: xsltFileUpgrader
+            path: *permission-mappings-config
+            template: crafter/studio/upgrade/3.1.x/3.1.0.29/site/permission-mappings-config.xslt
```

Comment:
Maybe we could place these files in a `3.1.0.29/config` folder to make it clear that they will not change site content

Reply:
`config` folder is part of the site content. Most of the changes we have in upgrade pipeline are on config files

Reply:
`config` is outside of `site`, no?

Or are you saying that anything in the BP is `site`?

Reply:
`config` is outside of `site`, but in terms of structure it is inside site repository. maybe we don't render that content but it belongs to the site

Reply:
right, it is part of the repository but it doesn't contain any authored data, that is why I was suggesting to name the folder different

Reply:
all 28 previous upgrades have `site` and `system` only. just following naming convention already in place

Reply:
We can separate into content vs config in the future. Right now most upgrades against a site are happening against config.